run-integration-tests

```diff
@@ -0,0 +1,15 @@
+#!/bin/sh
+
+set -o errexit
+trap 'echo "Aborting due to errexit on line $LINENO. Exit code: $?" >&2' ERR
+set -o errtrace
+set -o pipefail
+
+_report_destination=${ARTIFACTS:-/dev/null}
+
+pipenv install --dev
+pipenv run ./app.py
```

Comment:
```suggestion
MAIL_REPORT=1 pipenv run ./app.py
```

Can we keep the html mail report generation optional in python sources? I found it handy when developing or running integration tests locally. Otherwise, all the output is stored in HTML and stdout/stderr does not provide any information.

Reply:
pls chk latest commit